common/src/main/java/org/dash/wallet/common/Configuration.java

```diff
@@ -92,9 +91,7 @@ public class Configuration {
     private static final int PREFS_DEFAULT_BTC_SHIFT = 0;
     private static final int PREFS_DEFAULT_BTC_PRECISION = 8;
     public static final String PREFS_KEY_IS_DASH_TO_FIAT_DIRECTION = "is_dash_to_fiat_direction";
-    public static final String PREFS_KEY_SEND_PAYMENT_EXCHANGE_CURRENCY = "send_payment_exchange_currency";
-    public static final String PREFS_KEY_DEFAULT_FIAT_CURRENCY_CHANGED = "fiat_currency_changed";
-    public static final String PREFS_KEY_CURRENT_FIAT_CURRENCY_CHANGED = "current_fiat_currency_changed";
```

Comment:
This prefs are not used anymore.

Reply:
I suppose if we resurrect bluetooth support, then we can go into the history and find the missing pieces.